Makefile

```diff
@@ -12,6 +14,7 @@ login-sandbox:
 	fly -t sandbox login  --concourse-url http://192.168.100.4:8080/
 
 pipeline-check:
-	yes | fly -t ${TARGET} set-pipeline -c fissile-stemcell-openSUSE-check.yml -p fissile-stemcell-openSUSE-check -l ../cloudfoundry/secure/concourse-secrets.yml
+	gpg --decrypt --batch --no-tty ${CONCOURSE_SECRETS_FILE} > ${TMP_SECRETS_FILE} 2> /dev/null
+	yes | fly -t ${TARGET} set-pipeline -c fissile-stemcell-openSUSE-check.yml -p fissile-stemcell-openSUSE-check -l ${TMP_SECRETS_FILE}
 	fly -t ${TARGET} unpause-pipeline -p fissile-stemcell-openSUSE-check
-
+	rm -f ${TMP_SECRETS_FILE}
```

Comment:
This means that if we had a problem with `fly` (e.g. intermittent connection issues) we'll leave a decrypted secrets file on disk (because make aborts after first error). That doesn't sound very awesome.